legend-engine-executionPlan-execution-store-relational/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/test/full/functions/in/RedshiftTest.java

```diff
@@ -0,0 +1,288 @@
+package org.finos.legend.engine.plan.execution.stores.relational.test.full.functions.in;
```

Comment:
This does not seem to be a good place for this test
Also to be consistent with other db types. Can you add a test in TestGeneration.java